django/db/migrations/loader.py

```diff
@@ -6,11 +6,15 @@
 
 from django.apps import apps
 from django.conf import settings
+from django.db import connection as connection_
```

Comment:
@shaib, this is your reported ticket -- what's your take on how it should work? Seems to me maybe the check in the executor is more appropriate after all.